packages/api/db/migration/20230714100652_add_management_plan_group.js

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright (c) 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+export const up = async function (knex) {
+  await knex.schema.createTable('management_plan_group', (t) => {
+    t.uuid('management_plan_group_id').primary().defaultTo(knex.raw('uuid_generate_v4()'));
+    t.string('created_by_user_id').references('user_id').inTable('users');
+    t.string('updated_by_user_id').references('user_id').inTable('users');
```

Comment:
Should these two be notNullable also?

Reply:
None of the current `created_by_user_id` nor `updated_by_user_id` have a `NOT NULL` constraint. This PR is just _consistent_ with the rest, e.g.
```
/packages/api/db/migration$ rg "string\('created_by_user_id'\)" --type js

20210224135647_position_refactor.js
8:    t.string('created_by_user_id').references('user_id').inTable('users');
210:    t.string('created_by_user_id').references('user_id').inTable('users');

20210805200615_products.js
12:    t.string('created_by_user_id').references('user_id').inTable('users').defaultTo('1');

20210614122231_documents-structure.js
18:    t.string('created_by_user_id').references('user_id').inTable('users');

20221213065402_alter_irrigation_types_table.js
3:    table.string('created_by_user_id').references('user_id').inTable('users').defaultTo('1');

20211122153642_organic_history.js
10:    t.string('created_by_user_id').references('user_id').inTable('users');

20201124110018_activityLog_traceability.js
6:        t.string('created_by_user_id').references('user_id').inTable('users').defaultTo('1');

20201124123610_crop_farm_field_traceability.js
18:      t.string('created_by_user_id').references('user_id').inTable('users').defaultTo('1');

20230714100652_add_management_plan_group.js
19:    t.string('created_by_user_id').references('user_id').inTable('users');

20201101141537_organic_certifier_survey.js
21:      table.string('created_by_user_id').references('user_id').inTable('users');

20201222100509_support_ticket.js
15:      table.string('created_by_user_id').references('user_id').inTable('users');

20210210092322_sale-shift-task-base-attribute-fields.js
4:      t.string('created_by_user_id').references('user_id').inTable('users').defaultTo('1');
10:      t.string('created_by_user_id').references('user_id').inTable('users').defaultTo('1');

20230111164146_create_nomination_tables.js
8:    table.string('created_by_user_id').references('user_id').inTable('users').notNullable();
22:    table.string('created_by_user_id').references('user_id').inTable('users').notNullable();
37:    table.string('created_by_user_id').references('user_id').inTable('users').notNullable();
52:    table.string('created_by_user_id').references('user_id').inTable('users').notNullable();

20210511012038_crop_variety_table.js
8:    t.string('created_by_user_id').references('user_id').inTable('users');

20220207164158_notification.js
55:    table.string('created_by_user_id').references('user_id').inTable('users');
68:    table.string('created_by_user_id').references('user_id').inTable('users');

20221213065340_alter_location_defaults_table.js
5:    table.string('created_by_user_id').references('user_id').inTable('users').defaultTo('1');

20221114230005_alter_field_work_task_table.js
65:    t.string('created_by_user_id').references('user_id').inTable('users').defaultTo('1');
85:    t.string('created_by_user_id').references('user_id').inTable('users').defaultTo('1');
```

Reply:
Got it 👍